tensorflow/core/framework/kernel_def.proto

```diff
@@ -40,6 +40,12 @@ message KernelDef {
   // priority is 0. The higher the priority the better. By default (i.e. if
   // this is not set), we prefer GPU kernels over CPU.
   int32 priority = 6;
+
+  // Sub-type of device this kernel runs on.
+  // For first-party devices, subdevice_type is the same as device type.
+  // For PluggableDevices(third-party devices), subdevice_type is registered 
+  // by plugin.
+  string subdevice_type = 7;
```

Comment:
(for @tensorflow/api-owners )
Can we move this closer to the device_type field? (keeping the same number)

Reply:
@jzhoulon Could you please apply this change? Thank you!

Reply:
Thanks for the comments and reminder, I have changed it. please review it. thanks.